1/README.md

```diff
@@ -57,18 +57,16 @@ https://www.elettra.trieste.it/ICALEPCS99/proceedings/papers/wa2i01.pdf
 
 Additionally, it aims to:
 
-* Provide ... 
+* Provide an Open Source software toolkit for building distirbuted object oriented control systems. It should be freely available for all applications including industry. Tango Controls is sufficiently complete to control any small or large installation. Provide the necessary libraries for 3 programming languages which allow hardware to be interfaced and modelled in a device object model. Provide the tools for configuring, managing, monitoring and archiving large numbers of control points implemented as device attributes. Provide naming, browsing and hierarchies of devices and groups of devices.
```

Comment:
```suggestion
* Provide an Open Source software toolkit for building distributed object oriented control systems. It should be freely available for all applications including industry. Tango Controls is sufficiently complete to control any small or large installation. Provide the necessary libraries for 3 programming languages which allow hardware to be interfaced and modeled in a device object model. Provide the tools for configuring, managing, monitoring and archiving large numbers of control points implemented as device attributes. Provide naming, browsing and hierarchies of devices and groups of devices.
```

Reply:
I suggested 2 typos fixes (distributed and modeled)... Not so easy to see with github interface it seems.
I think you can put 1 sentence per line, it will be easier to suggest changes. If you don't end up your lines with 2 spaces, it will result in the same presentation.